app/main/views/direct_award.py

```diff
@@ -181,7 +198,8 @@ def create_project_search(project_id):
         audit_type=AuditTypes.create_project_search,
         user=updater_json['updated_by'],
         data={
-            'projectId': project_id,
+            'projectId': project.id,
+            'projectPublicId': project.public_id,
```

Comment:
Would the idea be for the id key always be referred to externally as the "public id" or for this to just be a transitional key?

It'd be 'orrible if we ended up with all of our `id` fields being `publicId`.

Reply:
External of the API, the public ID should just be 'the' ID of the resource. Internal to the API, including AuditEvents, we probably need to keep a distinction between the ID and the public/external ID.

Reply:
Oh yes duh I wasn't looking at the context of this